app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -311,4 +311,7 @@
     <string name="settings_column_count_5">5</string>
     <string name="settings_column_count_6">6</string>
     <string name="settings_column_count_7">7</string>
+    <string name="unsupportedFile">Este arquivo não é suportado</string>
+    <string name="addFromPkpass">Selecionar um arquivo do gerenciador de senhas (.pkpass)</string>
```

Comment:
@AsmodeumX Please note that this is "Passbook", not "password manager": https://en.wikipedia.org/wiki/PKPASS